src/Components/pages/Book.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from './button';
+
+const Book = ({ title, author }) => (
+  <div className="bookcontainer">
+    <div className="book">
+      <div className="details">
+        <h3>{title}</h3>
+        <p className="author">{author}</p>
+      </div>
+      <Button color="red" text="Remove" />
+    </div>
+    <Button color="#0290ff" text="UPDATE PROGRESS" />
+  </div>
```

Comment:
- [ ] As this represents an element of a list, it makes sense to use the `<li>` tag as the main container. This will provide better semantic value to your page! 🤓 🔧 

```suggestion
  <li className="bookcontainer">
    <div className="book">
      <div className="details">
        <h3>{title}</h3>
        <p className="author">{author}</p>
      </div>
      <Button color="red" text="Remove" />
    </div>
    <Button color="#0290ff" text="UPDATE PROGRESS" />
  </li>
```